src/app/reducers/templates/redux/template.effects.ts

```diff
@@ -171,7 +169,7 @@ export class TemplateEffects {
   private onNotify(template: any, message: string) {
     this.notificationService.message({
       translationToken: message,
-      interpolateParams: { name: template.name }
+      interpolateParams: { name: truncate(template.name, 40) }
```

Comment:
Just check if truncated names appearing in the middle of the translation is fine

Reply:
done